src/conchat.js

```diff
@@ -18,11 +22,34 @@ class Con {
     );
   }
 
+  #sendMessage(collectionName, messageContent) {
+    set(ref(this.#database, `chats/${collectionName}`), {
+      messageContent,
+    });
+  }
+
+  #listenForMessages(collectionName) {
+    if (this.#state) {
+      const databaseRef = ref(this.#database, `chats/${collectionName}`);
+
+      onValue(databaseRef, (snapshot) => {
+        const data = snapshot.val();
```

Comment:
Realtime Database 구조를 확인해보니 chats/messages/messageContent 구조로 가져가고 있는 것 같은데 data라는 상수명을 좀 더 명확히 messages로 하는 것은 어떨까요?
그럼 더 database 구조를 명확히 파악할 수 있을 것 같아요!
```suggestion
        const messages = snapshot.val();
```